### ISSUE TITLE
Bug: Using same name notes in different projects (and kanban) causes to all projects on lonform tab disappear

### ISSUE THREAD

**Issue by yungaro** [kevboh/longform#197]
**Versions**
Longform version: 2.0.3
Obsidian version: 1.4.13
OS [e.g. macOS, Windows, iOS, Android]: Windows 10
Theme: Github Theme
Other plugins that you think might be relevant here:

**Describe the bug**
When I create two longform projects and create scenes with the same name in both projects, the longform tab doesnt show any project, I have to rename the notes in one of the projects or they wouldnt show in longform tab, it always happens when using Kanban with those notes.

**To Reproduce**
1. Create 2 projects in longform
2. Create scenes in both projects with same names
3. Add either projects notes to a kanban

**Expected behavior**
I wanted to have tracking of notes using longform for writting and kanban for organizing, but seems that using same name notes tends to weird behaviors on longform and kanban, a compatibility fix would be nice

Thanks

**Comment by yungaro** [comment]
I found that when you put a kanban inside a longform project, then longform stop loading projects

**Comment by b-camphart** [comment]
Can you share a link to the exact kanban plugin you're using?

**Comment by jgharding** [comment]
Yes I also checked the developer console when having a similar issue and it flagged “Kanban”. For now I’ve moved Kanbans and out of the folder.
